source/linux/How_to_Guides/FAQ/How_to_Check_Device_Tree_Info.rst

```diff
@@ -9,81 +9,152 @@ using the entries in /proc/device-tree.
 In the command prompt of Linux, you should see something similar to the
 following in the /proc/device-tree directory:
 
-::
-
-    root@j721e-evm:~# ls /proc/device-tree/
-    #address-cells  firmware                l3-cache0
-    #size-cells     fixedregulator-evm12v0  main_r5fss_cpsw9g_virt_mac0
-    __symbols__     fixedregulator-sd       memory@80000000
-    aliases         fixedregulator-vsys3v3  model
-    chosen          fixedregulator-vsys5v0  name
-    compatible      gpio-keys               pmu
-    connector       interconnect@100000     reserved-memory
-    cpus            interrupt-parent        serial-number
-    dma_buf_phys    ion                     sound@0
-    dummy-panel     l2-cache0               timer-cl0-cpu0
+.. code-block:: console
+
+	root@<machine>:~# ls /proc/device-tree/
```

Comment:
Excessive indentation. This code block is also not device agnostic. Any examples referencing the device tree or device tree sysfs entries will inherently be device specific. You don't need an ifconfig for each device, but if you don't tell the reader what you're using they may incorrectly assume the output should match theirs.

This comment applies to all code-blocks in this PR.

Reply:
Checkout https://github.com/TexasInstruments/processor-sdk-doc/blob/master/CONTRIBUTING.md#indentation-and-whitespace for details

Reply:
Will fix indentation. Agreed that device tree sysfs  entries will be device specific. And even though my requirement requests to make the logs more "generic", the best I could do is to provide logs using am62x as an example when possible. In this case am62x logs for devices: am62x, am64x, am62px, and am62ax, should be good enough since the devices are similar to each other. ATM I think using j721e logs as examples is causing confusion...